filebeat/module/iis/w3svc/test/test.log

```diff
@@ -0,0 +1,3 @@
+#Fields: date time s-ip cs-method cs-uri-stem cs-uri-query s-port cs-username c-ip cs(User-Agent) sc-status sc-substatus sc-win32-status time-taken
```

Comment:
What is happening when this first line is ingested as it is not part of the pattern?

Reply:
It should be ignored as it is commented out.

Reply:
How is this ignored? I probably miss something here but filebeat reads line by line and does not really "interpret" if something is commented out.

Reply:
I though this was implicit in the config.
I added this line to fix your concern:
` exclude_lines: ["^#"]`